src/main/resources/main.fxml

```diff
@@ -30,8 +32,17 @@
                             </content>
                         </ScrollPane>
                     </items>
-                </SplitPane>
-                <fx:include fx:id="attribute" source="Attribute.fxml"/>
+            </SplitPane>
+            <HBox prefHeight="100.0" prefWidth="200.0">
+               <children>
+                  <fx:include fx:id="attribute" source="Attribute.fxml" />
+                  <BorderPane prefHeight="47.0" prefWidth="139.0" HBox.hgrow="ALWAYS">
```

Comment:
why is it that the borderpane needs to be Hbox.hgrow=ALWAYS ?? this shouldnt be necessary since attribute bar has the always grow factor..

I am not sure, can you test it ?

Reply:
He needs it to fill in the rest of the bottom of the window, attribute doesn`t have the HBox.hgrow=ALWAYS.